camp_2/README.md

```diff
@@ -222,7 +222,7 @@ For deploying a contract using Protostar, we will follow the steps we learned in
 1. Deploy an account. Notice the devnet already deployed accounts for us.
 2. Compile the contract.
 3. Declare the contract.
-4. Invoke the function `deployContract` of the UDC. However, at the end of this section you will learn how not to invoke the UDC and simply use Protostar's `deploy`.
+4. Invoke the function `deployContract` of the UDC. However, at the end of this section you will learn how not to invoke the UDC directly and simply use Protostar's `deploy`.
```

Comment:
"...at the end of this section you will learn **how not to** invoke the UDC..." I find this wording confusing (I know it's not part of your proposed changes but I think it's a good time to review it)

Reply:
I agree, changed it to:

`4. Invoke the function `deployContract` of the UDC to deploy. Also, at the end of this section we will learn how to simply use Protostar's `deploy`.
`